### PR TITLE
Include Facts

### PR DESCRIPTION
**Pull request by chrushr**
Include facts for cities. Rhode Island doesn't have a fact because we are not sure about using the entire state.

### CODE REVIEW

File: src/data/data.js [hunk 1]
```diff
@@ -914,14 +1016,16 @@ const data = [
...
       'roadConnections':0.8,
       'roadRelation':0.0018331810000000001,
       'roadTag':0.19,
       'roundabout':0.0,
-      'score':0.047580000000000004
-  }
- 
+      'score':0.047580000000000004,
+      'factName':'Fun Demographic Fact',
+      'fact':" This city is the second-most ractially diverse city in the US. The city's Branck Brook Park is home to the largest collection of cherry blossom trees (5,000)/"
```
Comment (savannahostrowski) [inline src/data/data.js]: Typo in word racially.

File: src/data/data.js [hunk 2]
```diff
@@ -806,12 +896,14 @@ const data = [
...
-      'thumbnail': 'images/thumbnails_cities/CT_Hartford.jpg',
+      'thumbnail':'images/thumbnails_cities/CT_Hartford.jpg',
       'roadConnections':0.88,
       'roadRelation':0.0,
       'roadTag':0.12,
       'roundabout':0.00074239,
-      'score':0.0251
+      'score':0.0251,
+      'factName':'Fun Fact',
+      'fact':' This city is nickmamed "Insuranc Capital of the World" as it hosts many insurance company headquarters and is the regions major industry. '
```
Comment (savannahostrowski) [inline src/data/data.js]: Typo in nicknamed and Insurance.

File: src/data/data.js [hunk 3]
```diff
@@ -788,12 +876,14 @@ const data = [
...
-      'thumbnail': 'images/thumbnails_cities/DC_Washington.jpg',
+      'thumbnail':'images/thumbnails_cities/DC_Washington.jpg',
       'roadConnections':0.89,
       'roadRelation':0.025641026,
       'roadTag':0.08,
       'roundabout':0.01007326,
-      'score':0.02295
+      'score':0.02295,
+      'factName':'Fun Fact',
+      'fact':' This city is 19% parks by area (9,122 acres) whicm makes it the second-highest percentage of parks among high-density US cities. '
```
Comment (savannahostrowski) [inline src/data/data.js]: Typo in which

File: src/data/data.js [hunk 4]
```diff
@@ -698,12 +776,14 @@ const data = [
...
-      'thumbnail': 'images/thumbnails_cities/AR_LittleRock_AR.jpg',
+      'thumbnail':'images/thumbnails_cities/AR_LittleRock_AR.jpg',
       'roadConnections':0.77,
       'roadRelation':0.002306805,
       'roadTag':0.23,
       'roundabout':0.0034602079999999998,
-      'score':0.018359999999999998
+      'score':0.018359999999999998,
+      'factName':'Fun Fact',
+      'fact':' This city is home to the American Taekwondo Association World Headquarters as well as several other Taekwondo organizations. '
```
Comment (savannahostrowski) [inline src/data/data.js]: Remove space from beginning of fact.

File: src/data/data.js [hunk 5]
```diff
@@ -680,12 +755,15 @@ const data = [
...
-      'thumbnail': 'images/thumbnails_cities/CA_SF.jpg',
+      'thumbnail':'images/thumbnails_cities/CA_SF.jpg',
       'roadConnections':0.95,
       'roadRelation':0.014096186,
       'roadTag':0.03,
       'roundabout':0.004145937,
-      'score':0.018009999999999998
+      'score':0.018009999999999998,
+      'factName':'Fun Fact',
+      'fact':' Human habitation of this city dates to 3,000 BC and was an important center for the Spanish and Mexican empires. '
```
Comment (savannahostrowski) [inline src/data/data.js]: Remove space from beginning of fact.

File: src/data/data.js [hunk 6]
```diff
@@ -626,12 +695,14 @@ const data = [
...
-      'thumbnail': 'images/thumbnails_cities/ME_Portland.jpg',
+      'thumbnail':'images/thumbnails_cities/ME_Portland.jpg',
       'roadConnections':0.82,
       'roadRelation':0.006993007,
       'roadTag':0.17,
       'roundabout':0.001748252,
-      'score':0.01697
+      'score':0.01697,
+      'factName':'Fun Fact',
+      'fact':" This city is built upon the Great Neck peninsula which juts out into Casco Bay. It served as Maine's capital city from 1820 to 1832. "
```
Comment (savannahostrowski) [inline src/data/data.js]: Stick to single quotes everywhere please!

File: src/data/data.js [hunk 7]
```diff
@@ -536,12 +595,14 @@ const data = [
...
-      'thumbnail': 'images/thumbnails_cities/WY_Cheyenne.jpg',
+      'thumbnail':'images/thumbnails_cities/WY_Cheyenne.jpg',
       'roadConnections':0.68,
       'roadRelation':0.017751479,
       'roadTag':0.3,
       'roundabout':0.0,
-      'score':0.015040000000000001
+      'score':0.015040000000000001,
+      'factName':'Fun Geography Fact',
+      'fact':' This city, located in the southeastern corner of Wyoming just 9 miles from the Colorado border, is the least centrally located state capitals.   '
```
Comment (savannahostrowski) [inline src/data/data.js]: Remove spaces at beginning and end

File: src/data/data.js [hunk 8]
```diff
@@ -121,13 +134,14 @@ const data = [
...
-
+      'thumbnail':'images/thumbnails_cities/NV_LV.jpg',
       'roadConnections':0.85,
       'roadRelation':0.00596853,
       'roadTag':0.14,
       'roundabout':0.00596853,
-      'score':0.00946
+      'score':0.00946,
+      'factName':'Fun Geography Fact',
+      'fact':' This city sits on the "floor" of the Mojave Desert. On average, the temperate exceeds 100 degrees F on 74 days during the year. '
```
Comment (savannahostrowski) [inline src/data/data.js]: Use `&deg;` for degrees symbol.

File: src/data/data.js [hunk 9]
```diff
@@ -83,13 +93,15 @@ const data = [
...
-
+      'thumbnail':'images/thumbnails_cities/MT_Billings.jpg',
       'roadConnections':0.76,
       'roadRelation':0.0,
       'roadTag':0.24,
       'roundabout':0.0,
-      'score':0.00926
+      'score':0.00926,
+      'factName':'Fun History Fact',
+      'fact':" A nearby landmakr, Pompey's Pillar, was named by William Clark (of Lewis and Clark fame) after the son of Sacajawea. "
```
Comment (savannahostrowski) [inline src/data/data.js]: Typo in landmark.

**Comment by chrushr** [comment]
Made the changes above and corrected typos/grammar mistakes.

**Pull request merged**